README.md

```diff
@@ -3,20 +3,37 @@
 This provides a standard GitHub Action to upload nightly builds to the
 scientific-python nightly channel.
 
-In your Continuous Intregration pipeline once you've built you wheel, you can
+In your Continuous Integration pipeline once you've built you wheel, you can
 use the following snippet to upload to our central nightly repository:
 
+<!-- c.f. https://github.com/scientific-python/upload-nightly-action/pull/13 and
+https://github.com/matplotlib/matplotlib/pull/26023#discussion_r1212539700
+for short summary of why using commit SHA -->
+
```

Comment:
@bsipocz ...and after a long delay given too many meetings and weekend travel this is in now. Look good?